driver.go

```diff
@@ -36,7 +35,7 @@ func (d ipfsDriver) Create(r volume.Request) volume.Response {
 		return volume.Response{}
 	}
 
-	volumePath := filepath.Join(d.mountPoint, volumeName)
+	volumePath := volumeName
```

Comment:
or `docker run -it --rm -v ipns:Qmdi… --volume-driver=ipfs …`, `docker run -it --rm -v ipfs:Q…`